[8.0] fix travis build

gurneyalex opened the pull request:
- update travis config, add transifex support
- fix stock_reserve test, which would fail when run at module install time, 
  because the test would run before the parent left/right of stock.location was 
  updated, which killed the reserved stock computation. 

closes #9 #67

gurneyalex committed:
update travis config, setup transifex

gurneyalex committed:
force recomputation of stock.location hierachy

before running the tests. This fixes the failure experienced when running the
test in --init mode rather than --update mode.

pedrobaeza commented:
Is there any possibility to join all tests or stock_reserve still breaks the other tests?

laetitia-gangloff commented:
Is it link to this issue : https://github.com/OCA/stock-logistics-warehouse/issues/67 ?

gurneyalex commented:
@pedrobaeza trust me on that, if I split, it is because it is needed

gurneyalex commented:
@laetitia-gangloff yes

pedrobaeza commented:
hehe, OK, waiting for runbot result to fast-track to have green ASAP. Check also this issue about coverage: https://github.com/OCA/maintainer-quality-tools/issues/247

lepistone commented:
:+1: thanks

gurneyalex commented:
@pedrobaeza double checking the test split

gurneyalex commented:
no idea about the coverage issue

clonedagain commented on stock_reserve/test/stock_reserve.yml:
```diff
@@ -1,4 +1,9 @@
+  I force recomputation of stock.location parent left/right
+-
+  !python {model: stock.location}:
+    self._parent_store_compute(cr)
```
Can you explain why we need to do it ourself in the test description please?

lepistone replied:
+1

gurneyalex commented:
@pedrobaeza wait a sec for the merge plz

gurneyalex commented:
@clonedagain is the commit message not enough ?

pedrobaeza commented:
OK, waiting till you tell me

gurneyalex commented:
@pedrobaeza forcing the parent recomputation in stock_reserve_sale tests allows to not split the tests. Updated, rebased, pushed. Let's confirm that travis is green and wait until @clonedagain confirm he is ok with the patch.

gurneyalex commented:
@pedrobaeza covered @clonedagain  and @lepistone's request. OK for merge from my side.

lepistone commented:
thanks

pedrobaeza commented:
Merging!

clonedagain commented:
@gurneyalex thanks for taking time to explain the problem and the fix.
